### ISSUE TITLE
Number of hospitals in the dataset

### ISSUE THREAD

**Issue by theonesp** [MIT-LCP/eicu-website#40]
The number of hospitals in the 'hospital' table is 459, nevertheless, there is no way to join the hospitals from this table with actual patients since this table only has 4 columns: hospitalid, numbedscategory, teachingstatus, region.
 
If we look in the 'patient' table and then the field hospitalid, it goes up to 364 unique hospitals.

**Comment by jraffa** [comment]
Yes, some hospitals are not available in the public subset for a variety of reasons.

There should be no problem with joining the hospitalid in this table to the patient table.  But perhaps I am misunderstanding something.

**Comment by alistairewj** [comment]
I think it's more there is no point in having these hospitals in the table since they don't exist in the data.

**Comment by theonesp** [comment]
I agree, it's confusing having different number of total hospitals
depending on the table you are looking at, specially if some of them don't
exist in the rest of the data.

Miguel

------------------------------

  [image: MIT][image: imes]

*Miguel Ángel Armengol de la Hoz **✉* <maarmeng@mit.edu> / Senior Research
Associate
*MIT - Institute for Medical Engineering and Science *
*Laboratory for Computational Physiology*
MIT, E25-235- 53 Carleton St.
Cambridge, MA 02139

[image: LinkedIn logo]
<https://www.linkedin.com/in/miguelangelarmengol/?locale=en_US>
